calico-enterprise_versioned_docs/version-3.18/releases.json

```diff
@@ -72,14 +339,14 @@
         "version": "v3.18.0"
       },
       "eck-kibana": {
-        "version": "7.17.11"
+        "version": "7.17.13"
       },
       "kibana": {
         "image": "tigera/kibana",
         "version": "v3.18.0"
       },
       "eck-elasticsearch": {
-        "version": "7.17.11"
+        "version": "7.17.13"
```

Comment:
I think this is the wrong one. It should be the one if you scroll up.

Reply:
Fixed.